test/e2e/semantic-domains.spec.ts

```diff
@@ -86,66 +67,28 @@ test.describe('Lexicon E2E Semantic Domains Lazy Load', () => {
     await expect(editorPage.senseCard.locator(editorPage.semanticDomainSelector).first()).toHaveText(semanticDomain1dot1English);
 
     // can change Project default language back to Thai
-    await projectSettingsPage.gotoProjectSettingsDirectly(project.id, project.name);
-    await expect(projectSettingsPage.projectTab.defaultInterfaceLanguageInput.locator('option[selected="selected"]')).toHaveText('English');
-    await projectSettingsPage.page.waitForTimeout(1000);
-    await projectSettingsPage.projectTab.defaultInterfaceLanguageInput.selectOption({ label: 'ภาษาไทย - semantic domain only' });
-    await projectSettingsPage.page.waitForTimeout(2000);
-    await projectSettingsPage.projectTab.saveButton.click();
+    await projectSettingsPage.goto();
+    await projectSettingsPage.setDefaultInterfaceLanguage('ภาษาไทย - semantic domain only', 'English');
     await expectOptionSelectedInSelectElement(projectSettingsPage.projectTab.defaultInterfaceLanguageInput, 'ภาษาไทย');
 
-    // should be using Thai Semantic Domain after refresh
+    // should be using Thai Semantic Domain
     await editorPage.goto();
     await expect(editorPage.senseCard.locator(editorPage.semanticDomainSelector).first()).toHaveText(semanticDomain1dot1Thai);
-    // browser refresh
-    await editorPage.page.reload();
-    await expect(editorPage.senseCard.locator(editorPage.semanticDomainSelector).first()).toHaveText(semanticDomain1dot1Thai);
 
     // can change user interface language
     await expect(pageHeader.languageDropdownButton).toHaveText('ภาษาไทย');
     await pageHeader.languageDropdownButton.click();
     await pageHeader.languageDropdownItem.filter({ hasText: 'English' }).click();
     await expect(pageHeader.languageDropdownButton).toHaveText('English');
 
-    // should still have Thai for Project default language
-    await projectSettingsPage.gotoProjectSettingsDirectly(project.id, project.name);
-    await expect(projectSettingsPage.projectTab.defaultInterfaceLanguageInput.locator('option[selected="selected"]')).toHaveText('ภาษาไทย - semantic domain only');
-
     // should be using English Semantic Domain
-    await editorPage.goto();
-    await expect(editorPage.senseCard.locator(editorPage.semanticDomainSelector).first()).toHaveText(semanticDomain1dot1English);
-
-    // should be using English Semantic Domain after refresh
-    // browser refresh
-    await editorPage.page.reload();
     await expect(editorPage.senseCard.locator(editorPage.semanticDomainSelector).first()).toHaveText(semanticDomain1dot1English);
 
     // should still have Thai for Project default language
-    await projectSettingsPage.gotoProjectSettingsDirectly(project.id, project.name);
+    await projectSettingsPage.goto();
     await expect(projectSettingsPage.projectTab.defaultInterfaceLanguageInput.locator('option[selected="selected"]')).toHaveText('ภาษาไทย - semantic domain only');
 
-    // can change user interface language to English
```

Comment:
I chopped off the end of this test, because it seemed redundant (it still a bit redundant, but that's fine)